L/LLVM/LLVM_full@15/bundled/patches/0100-revert-sdk-requirement-tsan.patch

```diff
@@ -0,0 +1,33 @@
+From beb762862751560f5499aaf2a4487277f08003e4 Mon Sep 17 00:00:00 2001
+From: Valentin Churavy <v.churavy@gmail.com>
+Date: Sat, 30 Apr 2022 14:55:36 -0400
+Subject: [PATCH] [PATCH] Revert "[TSan] Add CMake check for minimal SDK
+ requirements on Darwin"
+
+This reverts commit 3bf3996cd4ef2d4898b32d4cef52d6549bbe6820.
+---
+ compiler-rt/lib/tsan/rtl/CMakeLists.txt | 8 --------
+ 1 file changed, 8 deletions(-)
+
+diff --git a/compiler-rt/lib/tsan/rtl/CMakeLists.txt b/compiler-rt/lib/tsan/rtl/CMakeLists.txt
+index 0da2b89fd807..1bf6f8b187c4 100644
+--- a/compiler-rt/lib/tsan/rtl/CMakeLists.txt
++++ b/compiler-rt/lib/tsan/rtl/CMakeLists.txt
+@@ -111,14 +111,6 @@ if("${CMAKE_C_FLAGS}" MATCHES "-Wno-(error=)?unused-command-line-argument")
+ endif()
+ 
+ if(APPLE)
+-  # Ideally we would check the SDK version for the actual platform we are
+-  # building for here.  To make our lifes easier we assume the host SDK setup is
+-  # sane and use the macOS SDK version as a proxy for aligned SDKs.
+-  find_darwin_sdk_version(macosx_sdk_version "macosx")
+-  if ("${macosx_sdk_version}" VERSION_LESS 10.12)
```

Comment:
Probably also this one can be removed

Reply:
Ugh, it turns out this is still necessary because for some reasons the build system resets the declared version of the SDK to 10.10
```
[18:43:34] CMake Warning at /workspace/srcdir/llvm-project/compiler-rt/cmake/Modules/CompilerRTDarwinUtils.cmake:48 (message):
[18:43:34]   Overriding macosx SDK version to 10.10
[18:43:34] Call Stack (most recent call first):
[18:43:34]   /workspace/srcdir/llvm-project/compiler-rt/cmake/Modules/CompilerRTDarwinUtils.cmake:130 (find_darwin_sdk_version)
[18:43:34]   /workspace/srcdir/llvm-project/compiler-rt/cmake/config-ix.cmake:481 (darwin_test_archs)
[18:43:34]   /workspace/srcdir/llvm-project/compiler-rt/CMakeLists.txt:268 (include)
```
and so this test still fails